js/run_examples.js

```diff
@@ -0,0 +1,159 @@
+$(document).ready(function()
+{
+    if ($('body')[0].id == "Home")
```

Comment:
`document.getElementsByTagName('body')[0].id == "Home"`

I'm all for jQuery for when it abstracts differences between browsers, but for the simple stuff let's stick with plain JS.